setup.py

```diff
@@ -46,8 +46,22 @@
 
     # List of dependencies
     install_requires=[
-        'cellmlmanip>=0.2.0',
-        'Jinja2>=2.11',
+        'decorator>=4.4.2, <4.5.0',
```

Comment:
I would say `<5`, and similarly for the others, so only a major version change is prevented.